### ISSUE TITLE
ETH gas transaction times out when trying to lend or borrow

### ISSUE THREAD

**Issue by jamesjames23** [UnFederalReserve/community#52]
**Describe the bug**
I went on the UnFed Lending page as soon as it went live. I managed to lend my ERSDL quite quickly.

I then went to lend Link. The gas transaction timed out a few times so I kept clicking and confirming gas costs after each transaction to go again. It eventually went through and I did the same with lending ETH which also took a few attempts.

I have since tried to allow my ERSDL to be collateral last night and today for borrowing. I’ve clicked on this and confirmed the gas fees, it just does the collateral transaction pending white box in the middle of the screen. After perhaps 5-10 minutes this disappears and it seams to have failed. It has failed about 13 times. I also tried to go back to lend more ETH but those transactions were also doing the same about 3 times. It doesn’t seem to let any of my ETH gas fees go through. I have tried multiple of those times paying the ‘fast’ gas fee.

website 
**To Reproduce**
1. Go to UnFed Lending website
2. Click on supply ETH. (I have it in my wallet but have already got some supplies)
3. Confirm gas fee
4. White loading transaction box eventually just disappears like the transaction has failed.

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS:
 - Browser- I use chrome
 

**Wallet Version (please complete the following information):**
 - Wallet: I’ve been using MetaMask 
 - Public Address: [e.g. 0x89205A3A3b2A69De6Dbf7f01ED13B2108B2c43e7]

**Additional context**

**Comment by hskainth** [comment]
@jamesjames23 we will look into this and get back to you.

**Comment by jamesjames23** [comment]
Hi. My problem is now solved. 

I had a backlog of failed transactions so I had to pay to cancel every single one of them, all 15 which was quite annoying. I was picking the highest gas option nearly every time so I don’t quite understand why it all went wrong. 

It would be good if the website flashed up a warning saying “do not try to issue another transaction until you have checked your first transaction has cleared or cancelled” after the first transaction timed out. I know of about 15-20 people that have had similar issues where they currently can’t get any transactions to go through on the website so I’m guessing that will be the same problem. I will pass the info on to them.

thanks
